src/commons/application/types/SessionTypes.ts

```diff
@@ -121,6 +123,7 @@ export type SessionState = {
   readonly assessmentOverviews?: AssessmentOverview[];
   readonly assessments: Map<number, Assessment>;
   readonly gradingOverviews?: GradingOverview[];
+  readonly gradingOverviewsTest?: GradingOverview[];
```

Comment:
Can we rename `gradingOverviewsTest` to something more appropriate? It's not very clear what this variable is for...